exercises/protein-translation/canonical-data.json

```diff
@@ -118,13 +118,13 @@
           "expected": ["Methionine","Phenylalanine","Tryptophan"]
         },
         {
-          "description": "Translation stops if STOP codon present 1",
+          "description": "Translation stops if STOP codon present in short string",
           "property": "translate",
           "strand": "AUGUUUUAA",
           "expected": ["Methionine","Phenylalanine"]
```

Comment:
`STOP` is included in the above test cases where it is the only codon; either the above cases should expected empty lists as a result or `STOP` should be included in the results here.